tests/phpunit/tests/media.php

```diff
@@ -4700,7 +4768,7 @@ function ( $content ) use ( &$result, $context ) {
 		);
 		apply_filters( 'the_content', '' );
 
-		$this->assertSame( array(), $result );
+		$this->assertSame( array( 'decoding' => 'async' ), $result );
```

Comment:
Per my other feedback above, this should still result in an empty array since no modifications should be made for the special contexts tested here.

Reply:
I see. Makes sense ;)